client/my-sites/upgrades/navigation.jsx

```diff
@@ -123,14 +118,12 @@ var UpgradesNavigation = React.createClass( {
 		var items;
 
 		if ( this.props.selectedSite.jetpack ) {
-			items = [ 'Plans', 'My Upgrades' ];
-		} else if ( config.isEnabled( 'upgrades/domain-management/list' ) ) {
-			items = [ 'Plans', 'Domains', 'Email', 'My Upgrades' ];
+			items = [ 'Plans', 'My Purchases' ];
+		} else {
+			items = [ 'Plans', 'Domains', 'Email', 'My Purchases' ];
 			if ( config.isEnabled( 'upgrades/purchases/list' ) ) {
```

Comment:
This is always true in production, and I think we want to link to `/purchases` here too, so we should probably just get rid of this block.